### ISSUE TITLE
uninstalled, but "custom games" still exists in the console

### ISSUE THREAD

**Issue by abe1007** [justMaku/gpghax#27]
Asian console(HK/JP). I tried the uninstall and found that the "Custom Game" still remains in the console, though the cover picture is missing. I'm not sure if the console has been totally restored. Any possibilities that something potentially risky is still in the NAND? How do I make it completely "clean" before I try the Bleemsync? Thanks.

![37b65fa10b8c1c4cbba579e3b993414](https://user-images.githubusercontent.com/25562328/49882929-95537d80-fe6c-11e8-8ef0-a977986f4fbf.jpg)
![c7ca6be9b1ee7cff543b5b09844da45](https://user-images.githubusercontent.com/25562328/49882934-9684aa80-fe6c-11e8-907b-008f31b76030.jpg)

**Comment by justMaku** [comment]
Did you follow the uninstall guide? If so, can you please attach the `uninstall.log` and `filelist.log`?

**Comment by abe1007** [comment]
Yes I followed every step, including replacement of "default.db" with the jp version (renamed of course). I've uploaded the uninstall.log and filelist.log.
BTW, after the "restoration" the original games seem to be fine, but the "Custom Game" icon is still there. Thank you for helping me out.

[filelist.log](https://github.com/justMaku/gpghax/files/2672676/filelist.log)
[uninstall.log](https://github.com/justMaku/gpghax/files/2672677/uninstall.log)

**Comment by justMaku** [comment]
I think I know what’s up, the Japanese DB in the repo still has the custom game entry, I will upload new version in few hours.

**Comment by abe1007** [comment]
Thanks. The Japanese console users will be grateful. :)

**Comment by zkf1976** [comment]
today, I tried the uninstall and found that the "Custom Game" still remains in the console too.
my Japanese console too

**Comment by justMaku** [comment]
I have updated the JP database, please try again with the newest version. Sorry for delay.

**Comment by zkf1976** [comment]
The problem has been solved! Thank you very much.

**Comment by abe1007** [comment]
Thanks so much for solving the problem, I'm very grateful.
